sqlmesh/cli/main.py

```diff
@@ -428,19 +429,27 @@ def diff(ctx: click.Context, environment: t.Optional[str] = None) -> None:
     help="Output text differences for the rendered versions of the models and standalone audits",
 )
 @opt.verbose
+@opt.very_verbose
 @click.pass_context
 @error_handler
 @cli_analytics
 def plan(
-    ctx: click.Context, verbose: bool, environment: t.Optional[str] = None, **kwargs: t.Any
+    ctx: click.Context,
+    verbose: bool,
+    very_verbose: bool,
```

Comment:
Let's not do this.  See a better example here: https://click.palletsprojects.com/en/stable/options/#counting. We want one argument, which when repeated results in higher verbosity levels.